tests/python/test_transformer_engine.py

```diff
@@ -21,6 +23,17 @@ class ComputeType(Enum):
     BACKWARD = auto()
 
 
+# This benchmark is instrumented with cudaProfilerStart/Stop. Therefore, one
+# can collect stats of a non-warmup benchmark iteration using
+#
+# ```bash
+# mpirun -np <processes> nsys profile --capture-range=cudaProfilerApi --capture-range-end=stop pytest tests/python/test_transformer_engine.py -k <filter> --only-mpi
+# ```
+#
+# Because of `--capture-range-end=stop`, the above command only profiles the
+# **first** iteration. This should be enough because timing changes very little
```

Comment:
Might want to change to capture at least 2 iterations. In general the first iteration can be significantly slower if the gpus start at different times because the allreduce synchronizes them and nsys profiling overhead.

Reply:
I meant to say the first non-warmup iteration. Is that good enough?

Reply:
No, this is separate from warmup. It's overhead caused from starting the profiler

Reply:
Done -- thanks!